tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1585,9 +1585,11 @@ class MklQuantizedConv2DOp
       // Re-scale bias if either of following 2 conditions are met:
       // 1. Bias is not const;
       // 2. Bias is const, but bias cache is empty (first iteration).
-      // TODO Disabling bias caching for now. The graph obtained after
-      // quantize_graph.py does not run with correct accuracy when this feature
-      // is enabled.
+
+      // TODO(intel-tf): Re-enable bias caching. Currently, the graph obtained
+      // after quantize_graph.py does not run with correct accuracy with this
+      // feature enabled.
+
```

Comment:
Nit: Please remove this blank line. I don't think we need this extra space between comment and code. (I only needed the blank line in line 1588 but not 1592.) Sorry about the nit!